phpunit/class-wp-theme-json-test.php

```diff
@@ -2071,4 +2071,27 @@ public function data_process_blocks_custom_css() {
 			),
 		);
 	}
+
+	public function test_internal_syntax_is_converted_to_css_variables() {
+		$result = new WP_Theme_JSON_Gutenberg(
+			array(
+				'version' => WP_Theme_JSON_Gutenberg::LATEST_SCHEMA,
+				'styles'  => array(
+					'blocks' => array(
+						'core/post-terms' => array(
+							'typography' => array( 'fontSize' => 'var(--wp--preset--font-size--small)' ),
+							'color' => array( 'background' => 'var:preset|color|secondary' ),
+						),
+					),
+				),
+			)
+		);
+		$styles = $result->get_data()['styles']['blocks']['core/post-terms'];
```

Comment:
We can use `get_raw_data` which simply returns the internal theme.json structure and nothing else while `get_data` does a lot of other things:

```suggestion
		$styles = $result->get_raw_data()['styles']['blocks']['core/post-terms'];
```